inst/include/Rcpp/sugar/functions/median.h

```diff
@@ -0,0 +1,297 @@
+// -*- mode: C++; c-indent-level: 4; c-basic-offset: 4; tab-width: 8 -*-
+//
+// median.h: Rcpp R/C++ interface class library -- median
+//
+// Copyright (C) 2012 - 2013 Dirk Eddelbuettel and Romain Francois
+// Copyright (C) 2016 - Nathan Russell 
```

Comment:
If there is one year only, no hyphen.  (Not that it matters but as I am in nitpick mode...)